controllers/storagecluster/cephcluster.go

```diff
@@ -1067,16 +1064,13 @@ func generateMonSpec(sc *ocsv1.StorageCluster, nodeCount int) rookCephv1.MonSpec
 
 func generateMgrSpec(sc *ocsv1.StorageCluster) rookCephv1.MgrSpec {
 	spec := rookCephv1.MgrSpec{
-		Count: getMgrCount(arbiterEnabled(sc)),
+		Count:                getMgrCount(),
+		AllowMultiplePerNode: statusutil.IsSingleNodeDeployment(),
```

Comment:
If it's a single node deployment, how about if we default to a single mgr instead of allowing multiple per node?

Reply:
Done